app/assets/stylesheets/home_template/css/bootstrap.min.css

```diff
@@ -0,0 +1,7327 @@
+/*!
```

Comment:
não já tem bootstrap? 😮

Reply:
Estou tirando os arquivos inúteis js, css ... já tirei bastante arquivos e estou filtrando os estilos o/ quanto ao bootstrap mantive o min porque o template está em na versão 4 vou remover ... para ver se algo quebra ...